### PR TITLE
Fix/acceptance tests

### PR DESCRIPTION
**Pull request by tohalla**

### CODE REVIEW

File: .github/workflows/acceptance_tests.yml [hunk 1]
```diff
@@ -0,0 +1,23 @@
+name: UpCloud Terraform provider acceptance tests
+
+on:
+  push:
+    branches:
+      - master
```
Comment (PopoSensei) [inline .github/workflows/acceptance_tests.yml:6]: Why not on PR:s?

Comment (tohalla) [reply .github/workflows/acceptance_tests.yml:6]: the acceptance tests take almost three hours to run 😁

Comment (tohalla) [reply .github/workflows/acceptance_tests.yml:6]: should probably look into that at some point as well

Comment (PopoSensei) [reply .github/workflows/acceptance_tests.yml:6]: Yep. I've been wondering about that too.

**Pull request merged**